docs/common-bugs.rst

```diff
@@ -0,0 +1,278 @@
+
+Issues with OSTree
+^^^^^^^^^^^^^^^^^^
+
+:Detection: The installation with the OSTree payload fails.
+:Solution: It might be related to the OSTree payload. Reassigning to Colin Walters.
+:Contact: `walters@redhat.com <mailto:walters@redhat.com>`_
+
+Failed to mount the install tree
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:Detection: The payload fails to set up and raises the error ``Failed to mount the install tree``.
+    This usually happens when Anaconda is unexpectedly terminated and started again. Some of
+    the Anaconda's mount points stays mounted and that causes the crash.
+:Example: `rhbz#1562239 <https://bugzilla.redhat.com/show_bug.cgi?id=1562239>`_
+
+System upgrades
+^^^^^^^^^^^^^^^
+
+:Detection: The system was upgraded, not installed.
+:Solution: Anaconda is not doing system upgrades. That is done by dnf-system-upgrade.
+    Reassigning to dnf.
+
+Storage issues
+--------------
+
+These issues are related to hardware, partitioning and storage configuration.
+
+Bug in blivet
+^^^^^^^^^^^^^
+
+:Detection: The exception starts in ``blivet`` or ``libblockdev``.
+:Solution: It seems to be an issue in the storage configuration library. Reassigning to blivet.
+:Example: `rhbz#1827254 <https://bugzilla.redhat.com/show_bug.cgi?id=1827254>`_
+
+Bug in blivet-gui
+^^^^^^^^^^^^^^^^^
+
+:Detection: The exception starts in ``blivet-gui`` or there is a problem with partitioning and
+    the reporter used Blivet-GUI as the partitioning method.
+:Solution: It seems to be an issue in blivet-gui. Reassigning.
+:Example: `rhbz#1833775 <https://bugzilla.redhat.com/show_bug.cgi?id=1833775>`_
+
+Failing hardware
+^^^^^^^^^^^^^^^^
+
+:Detection: The logs (journal or syslog) are full of kernel messages about I/O errors. For
+    example::
+
+        kernel: [sdb] tag#9 FAILED Result: hostbyte=DID_OK driverbyte=DRIVER_SENSE
+        kernel: [sdb] tag#9 Sense Key : Medium Error [current]
+        kernel: [sdb] tag#9 Add. Sense: Unrecovered read error - auto reallocate failed
+        kernel: [sdb] tag#9 CDB: Read(10) 28 00 1d 04 10 00 00 00 08 00
+        kernel: print_req_error: I/O error, dev sdb, sector 486805504
+
+:Solution: It looks like a hardware failure. Please, check your hardware.
+:Example: `rhbz#1633457 <https://bugzilla.redhat.com/show_bug.cgi?id=1633457>`_
+
+Unlocked LUKS
+^^^^^^^^^^^^^
+
+:Detection: The storage configuration fails with a message ``luks device not configured``.
+:Solution: Anaconda doesn't support LUKS devices that are unlocked outside the installer. The
+    device has to be unlocked in Anaconda.
+:Example: `rhbz#1624617 <https://bugzilla.redhat.com/show_bug.cgi?id=1624617>`_
+
+Cannot toggle container encryption
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:Detection: It is not possible to toggle the container encryption in the custom partitioning spoke.
+:Solution: Duplicate of the bug 1827254.
+:Example: `rhbz#1827254 <https://bugzilla.redhat.com/show_bug.cgi?id=1827254>`_
```

Comment:
I think we can remove this too. It's something which is showing up now because it was fixed recently but I don't think it have to be between knows bugs.

Reply:
Fixed.